src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  const fullPrice = days * 40;
+
+  if (days < 3) {
+    return fullPrice;
+  } else if (days < 7) {
+    return fullPrice - smallDiscount;
+  } else {
+    return fullPrice - bigDiscount;
+  }
```

Comment:
```suggestion
  if (days < 3) {
    return fullPrice;
  } else if (days < 7) {
    return (fullPrice - smallDiscount);
  }
  return (fullPrice - bigDiscount);
```
There's no need in last else. The function will return the last result if both previous checks won't pass.

For expressions use brackets. It's clearer for reading and understanding